config/rewrites.json

```diff
@@ -153,5 +153,147 @@
             "rewrite": false,
             "status": 302
         }
+    ],
+    "api.rackspace.com": [
+        {  "description": "Redirect root traffic to developer.rs.com/docs/",
+            "from": "^\\/$",
+            "to": "/docs/",
```

Comment:
Also, what you want to do is redirect paths from `http://api.rackspace.com/...` to `https://developer.rackspace.com/docs/...`, right? These rules just redirect to different paths on `api.rackspace.com`, which will be 404s. To do what you want, you'll need to rewrite the protocol and hostname by adding [`"toHost"` and `"toProtocol"` keys](https://github.com/annegentle/nexus-control/blob/api-redirects/config/rewrites.json#L113-L114), like this:

```
"toProtocol": "https",
"toHostname": "developer.rackspace.com",
```